clients/src/main/java/org/apache/kafka/clients/admin/internals/AlterConsumerGroupOffsetsHandler.java

```diff
@@ -141,9 +147,16 @@ private void handleError(
                 failed.put(groupId, error.exception());
                 break;
             case COORDINATOR_LOAD_IN_PROGRESS:
+                // If the coordinator is in the middle of loading, then we just need to retry
+                log.debug("OffsetCommit request for group {} failed because the coordinator" +
```

Comment:
Could we use the `apiName` function for the request in the logs? Would be good to change it here and in all the other handler classes.

Reply:
Good suggestion! Updated!